SpecializedServers/MultiSocks/DirtySocks/Messages/GseaIn.cs

```diff
@@ -49,6 +53,8 @@ public override void Process(AbstractDirtySockServer context, DirtySockClient cl
             else
             {
                 // TODO, send dirtysocks error.
+                client.SendMessage(new GseaOut());
+                return;
```

Comment:
There exist some error handling that exist. Ask me of you have a doubt.

Reply:
I never said I had any doubts, I know about the error handling in the Aries protocol, but this is entire unhandled in your implementation, but I'm covering all bases here in terms of packet responses.

Reply:
So, the error code is always materialized in form of a 4 char word appended to the request.

Example: gcremiss.

Look in the GQWK for an example.

 client.SendMessage(new MissOut(_Name));

Reply:
This will be addressed HOPEFULLY in rewrite